content/pages/health-care/index.md

```diff
@@ -51,8 +51,8 @@ With VA health care, you’re covered for regular checkups with your primary car
 
 <div class="va-alert usa-alert usa-alert-warning">
   <div class="usa-alert-body">
-  <h5 class="va-alert-title">Are you homeless or at risk of becoming homeless?<br><a id="crisis-expander-link">VA may be able to help</a>.
-  </h5>
+  <h4 class="va-alert-title">Are you homeless or at risk of becoming homeless?<br><a id="crisis-expander-link">VA may be able to help</a>.
```

Comment:
This one looks perfect from a padding perspective -- MST, substance abuse and PTSD could all also use more padding beneath the icon.